target/linux/ath79/dts/qca9558_openmesh_om5p-ac-v2.dts

```diff
@@ -7,26 +7,34 @@
 
 / {
 	compatible = "openmesh,om5p-ac-v2", "qca,qca9558";
-	model = "OpenMesh OM5P-AC V2";
+	model = "OpenMesh OM5P-AC v2";
```

Comment:
The commit message says you add a new device, but you alter some existing dts file here. COuld you please update the commit description.

Reply:
Please check the second paragraph of this PR.

There were "stub" files in OpenWrt but never actual support for OM5P-ACv2. This is also why there are people complaining about the not working OM5P-ACv2 images in OpenWrt 21.02

Reply:
Added a remark about this in the commit message.